src/components/CreatePlayer/CreatePlayer.js

```diff
@@ -0,0 +1,48 @@
+import React, { useContext, useState } from "react";
+import { Link } from "react-router-dom";
+import nanoid from "nanoid";
+
+import "./createPlayer.css";
+import Button from "../Button";
+import PlayerForm from "../PlayerForm";
+import { GameContext } from "../../contexts/gameContext";
+
+const CreatePlayer = props => {
+  const game = useContext(GameContext);
+  const [players, setPlayers] = game.usePlayers;
+  const [playerToEditId] = useState(null);
+
+  const handlePlayerCreate = player => {
+    const newData = [...players];
+
+    if (!player.id) {
```

Comment:
There will never be a player id since it's a new player, so I think this check can be removed.